### ISSUE TITLE
NameError: global name 'resolvconf_nameservers' is not defined

### ISSUE THREAD

**Issue by kostich** [sshuttle/sshuttle#24]
```
[user@box sshuttle] $ src/sshuttle --dns -r user@xx.xxx.xxx.xx:2201 0/0
Traceback (most recent call last):
  File "src/main.py", line 224, in <module>
    opt.syslog, opt.daemon, opt.pidfile)
  File "/home/user/sshuttle/src/client.py", line 703, in main
    ns_hosts += resolvconf_nameservers()
NameError: global name 'resolvconf_nameservers' is not defined
```

The commit is 0fb714893a5928cb4890c6027411525f9d824978. Fedora 22.

It seems that the commit d2ee34d71c0e81dcfe518f8784ecb45e4f4518a8 introduced this issue. If I do ```git checkout 3cf5002b62650c26a50e18af8d8c5c91d754bab9 --``` , I can use sshuttle with no problem.

**Comment by naclander** [comment]
Looks like the patch forgot to add resolvconf_nameservers to the list of functions imported from helpers.py.

However, just adding the function name doesn't help, because we then get the following error:
```
Traceback (most recent call last):
  File "./main.py", line 224, in <module>
    opt.syslog, opt.daemon, opt.pidfile)
  File "/home/hemite/Projects/sshuttle/src/client.py", line 703, in main
    ns_hosts += resolvconf_nameservers()
TypeError: unsupported operand type(s) for +=: 'NoneType' and 'list'
```

**Comment by brianmay** [comment]
There are a number of things I would like to do with sshuttle given the time, one of them being proper tests to pick up on this sort of issue.

It would appear that the this pull request #23 was incomplete and not properly tested. There seems to be some confusion over the type of the ns_hosts variable throughout the code: some places it is a list of strings, other places it is a list of tuples. In the above example, I believe the `ns_hosts` is a list of strings, but the `resolvconf_nameservers()` call returns a list of tuples.

Thinking of reverting the change until I have time to understand it a bit better.

**Comment by vieira** [comment]
Sorry for the issue caused by my previous pull request. I tested with and without ``--ns-hosts`` but failed to test with ``--dns``. 😞
This latest pull request will fix the issue (hopefully without introducing new ones). Please let me know if you happen to find any problems or have some suggestions.

**Comment by kostich** [comment]
@vieira Great! I've started testing out your pull request.

**Comment by brianmay** [comment]
I believe this should be fixed now by #25.

**Comment by kostich** [comment]
Yep, all is good now.
